gcp/gcp.go

```diff
@@ -1091,15 +1090,22 @@ func getDiskStatus(gDisk *compute.Disk) types.State {
 }
 
 //SQL_INSTANCE_STATE_UNSPECIFIED	The state of the instance is unknown.
-//RUNNABLE	The instance is running.
+//RUNNABLE	The instance is running, or has been stopped by owner.
 //SUSPENDED	The instance is currently offline, but it may run again in the future.
 //PENDING_DELETE	The instance is being deleted.
 //PENDING_CREATE	The instance is being created.
 //MAINTENANCE	The instance is down for maintenance.
 //FAILED	The instance failed to be created.
-func getDatabaseInstanceStatus(instance *sqladmin.DatabaseInstance) types.State {
+func getDatabaseInstanceStatus(instance *sqladmin.DatabaseInstance, operationsList *sqladmin.OperationsListResponse) types.State {
 	switch instance.State {
-	case "RUNNABLE", "PENDING_CREATE", "MAINTENANCE", "FAILED":
+	case "RUNNABLE":
+		// The sdk returns both running and stopped instances as RUNNABLE, so the most accurate way to tell if the instance is stopped is to check if the last operation was an update
+		lastOperation := operationsList.Items[0]
```

Comment:
This implementation depends on the behavior of some underlying system. I mean it is really clear that there is no sql instance without operations, but the result of the API call depends on a meaningful default `maxResults` on the server side. So code works but clean and should end in out of bound error.

Reply:
Thank you for the suggestions, it seems like their is a better approach: to check the instance.Settings.ActivationPolicy as it will be _NEVER_ in case of a stopped instance, and _ALWAYS_ for running instances
I will update the PR soon